<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SURV-I</string>
  </property>
  <property name="windowIcon">
   <iconset resource="surv-1.qrc">
    <normaloff>:/images/images/window-icon.png</normaloff>:/images/images/window-icon.png</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="actionStart">
         <property name="text">
          <string>Start</string>
         </property>
         <property name="icon">
          <iconset resource="surv-1.qrc">
           <normaloff>:/images/images/start.png</normaloff>:/images/images/start.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="actionStop">
         <property name="text">
          <string>Stop</string>
         </property>
         <property name="icon">
          <iconset resource="surv-1.qrc">
           <normaloff>:/images/images/stop.png</normaloff>:/images/images/stop.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="entitiesButton">
         <property name="text">
          <string>Entities</string>
         </property>
         <property name="icon">
          <iconset resource="surv-1.qrc">
           <normaloff>:/images/images/entities.png</normaloff>:/images/images/entities.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="showRegions">
         <property name="text">
          <string>Show Regions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showDetails">
         <property name="text">
          <string>Show Details</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="surv-1.qrc"/>
 </resources>
 <connections/>
</ui>
